core/ledger/kvledger/kv_ledger_provider.go

```diff
@@ -319,7 +319,7 @@ func (p *Provider) open(ledgerID string) (ledger.PeerLedger, error) {
 	p.collElgNotifier.registerListener(ledgerID, pvtdataStore)
 
 	// Get the versioned database (state database) for a chain/ledger
-	vDB, err := p.vdbProvider.GetDBHandle(ledgerID)
+	db, err := p.dbProvider.GetDBHandle(ledgerID)
```

Comment:
In the lines of above comment, `statedb` may be a better choice, if not `vDB`. Plain `db` looses the context.